src/shared/prompts.tsx

```diff
@@ -58,11 +65,12 @@ export async function confirm({
             </Button>
           </>
         }
+        initialFocus={cancelButton as RefObject<HTMLElement>}
```

Comment:
AAAAAARGH, `Ref*` typing. If there's an alternative, I'm all ears. Note that we can't use `useRef` here (have to use `createRef`) because this isn't technically a component function.

Reply:
Maybe one option is extracting the majority of the JSX to an actual component, where you can use `useRef`. Other than that, I think the type casting here is acceptable.